bindings/wasm/examples/worker.ts

```diff
@@ -457,8 +460,29 @@ async function exportGLB(manifold?: Manifold) {
     wrapper.addChild(node);
   }
 
+  const results: {[name: string]: string|null} = {};
+  if (manifold != null) {
+    const mesh = manifold.getMesh();
+    let vertices = new Float32Array(mesh.numVert * 3);
+    for (let i = 0; i < mesh.numVert; ++i) {
+      for (let j = 0; j < 3; ++j)
+        vertices[i * 3 + j] = mesh.vertProperties[i * mesh.numProp + j];
+    }
+    const model =
+        to3dmodel({simple: [{vertices, indices: mesh.triVerts, id: '1'}]});
+    const files: Zippable = {};
+    files['3D/3dmodel.model'] = strToU8(model);
+    files[fileForContentTypes.name] = strToU8(fileForContentTypes.content);
+    const zipFile = zipSync(files);
+    results['threeMFURL'] = URL.createObjectURL(
+        new Blob([zipFile], {type: 'application/octet-stream'}));
```

Comment:
I think this is the mimetype: [`application/vnd.ms-package.3dmanufacturing-3dmodel+xml`](http://www.iana.org/assignments/media-types/application/vnd.ms-package.3dmanufacturing-3dmodel+xml)